### ISSUE TITLE
[Bug] MaskRCNN with torchvision

### ISSUE THREAD

**Issue by isaacraf95** [open-mmlab/mmdeploy#1845]
### Checklist

- [ ] I have searched related issues but cannot get the expected help.
- [ ] 2. I have read the [FAQ documentation](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/faq.md) but cannot get the expected help.
- [ ] 3. The bug has not been fixed in the latest version.

### Describe the bug

I have trained a model with the following torchvision structure: 
`num_classes = len(CLASSES)
model = torchvision.models.detection.maskrcnn_resnet50_fpn(pretrained=True)
in_features = model.roi_heads.box_predictor.cls_score.in_features
model.roi_heads.box_predictor = FastRCNNPredictor(in_features, num_classes)
in_features_mask = model.roi_heads.mask_predictor.conv5_mask.in_channels
hidden_layer = 256
model.roi_heads.mask_predictor = MaskRCNNPredictor(in_features_mask,
                                                    hidden_layer,
                                                    num_classes)`

however, when I try to convert the model with the file mask_rcnn_r50_fpn_1x_coco.py I get a lot of warnings about model and loaded state dict do not match exactly, should I create a custom config? 

### Reproduction

`python mmdeploy/tools/deploy.py \
    mmdeploy/configs/mmdet/instance-seg/instance-seg_onnxruntime_static.py \
    mmdetection/configs/mask_rcnn/mask_rcnn_r50_fpn_1x_coco.py \
    checkpoints/model6_ckpt \
    mmdetection/demo/Pieza24_2.jpg \
    --work-dir mmdeploy_model/mask-rcnn \
    --device cpu \
    --dump-info`

### Environment

```Shell
python 3.8
```

**Comment by lvhan028** [comment]
`maks_rcnn_r50_fpn_1x_coco.py` is for mmdetection's `mask r-cnn`, which is different from torchvison's.

I am not sure if there is a way to convert the *.pth to mmdetction's, but you can  try `torch.onnx.export` to export the onnx model   directly

**Comment by isaacraf95** [comment]
oh, ok! I got it! my goal is really to convert torchvision model to tensorRT, do you know if there is a way?
